packages/manager/modules/pci/src/projects/project/storages/databases/add/add.utils.js

```diff
@@ -0,0 +1,95 @@
+export function getOrderDataFromModel(model) {
+  const orderData = {
+    description: model.name,
+    nodesPattern: {
+      flavor: model.flavor.name,
+      number: model.plan.nodesCount,
+      region: model.region.name,
+    },
+    plan: model.plan.name,
+    version: model.engine.selectedVersion.version,
+  };
+  if (model.usePrivateNetwork && model.subnet?.id?.length > 0) {
+    orderData.networkId = model.privateNetwork.regions?.find(
+      (region) => region.region === model.subnet?.ipPools[0].region,
```

Comment:
**nitpick:** Redundant safe-traversal operator. Already present in above `if` condition. However it can be added on `ipPools[0]`
```suggestion
      (region) => region.region === model.subnet.ipPools[0]?.region,
```

Reply:
Fixed :)